app/domain/sac_imports/roles/memberships_importer.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module SacImports::Roles
+  class MembershipsImporter < ImporterBase
+    BEITRAGSKATEGORIE_MAPPING = {
+      "Einzel" => :adult,
+      "Jugend" => :youth,
+      "Frei Fam" => :family,
+      "Familie" => :family,
+      "Frei Kind" => :family
+    }
+
+    SECTION_OR_ORTSGRUPPE_GROUP_TYPE_NAMES = [Group::Sektion.sti_name,
+      Group::Ortsgruppe.sti_name].freeze
+
+    def initialize(csv_source:, csv_report:, output: $stdout, failed_person_ids: [])
+      @rows_filter = {role: /^Mitglied \(Stammsektion\).+/}
+      super
```

Comment:
Ich find das ein wenig unschön, dass eine Instanzvariable gesetzt werden muss bevor `super` aufgerufen wird. Aber dafür muss man nicht alle parameter wiederholen beim super call:
```suggestion
      super(csv_source:, csv_report:, output:, failed_person_ids:, rows_filter: {role: /^Mitglied \(Stammsektion\).+/})
```

Reply:
:+1: